.github/renovate.json

```
@@ -10,6 +10,11 @@
       "matchFileNames": ["**/pom.xml"],
       "matchUpdateTypes": ["patch"],
       "automerge": true
+    },
+    {
+      "matchFileNames": ["\\.mise.toml$"],
```

`matchFileNames` on glob, ei regex: https://docs.renovatebot.com/configuration-options/#matchfilenames

---

Jos mä tekisin sit vaan?
```suggestion
      "matchFileNames": [".mise.toml"],
```

---

Ei riitä, lainatakseni dokumentaatiota:

> The following example matches package.json but not package/frontend/package.json:

    {
      "packageRules": [
        {
          "matchFileNames": ["package.json"],
          "labels": ["npm"]
        }
      ]
    }

---

Tai siis juu riittää jos .mise.toml on vain juuressa, kuten meillä taitaa olla. Muussa tapauksessa halutaan `**/.mise.toml`.

---

Eli varmaan tässä tapauksessa toi `[".mise.toml"]` toi riittää? Vai näetkö jotain syytä, että tuota pitäisi täsmentää?

---

Tuo riittää varmaankin nyt. 🙂